app/styles/join-section.module.css

```diff
@@ -0,0 +1,102 @@
+.join {
+  text-align: center;
+}
+
+.join__titleHighlighted {
+  color: var(--color-pinkl);
+  margin-top: -1.1rem;
```

Comment:
Can we please avoid using negative margins? it can lead to overlapping in elements and break the UI